backend/src/data_entry/validation.rs

```diff
@@ -525,8 +568,64 @@ impl Validate for PollingStationResults {
     }
 }
 
+fn validate_political_group_votes_errors(
```

Comment:
I would move this into a `impl CommonPollingStationResults { }` block (and then first argument will be `results: &self`)